api/v1/flow/README.md

```diff
@@ -1049,18 +1049,18 @@ This mirrors enum xlate_point in bpf/lib/trace_sock.h
 | Name | Number | Description |
 | ---- | ------ | ----------- |
 | UNKNOWN_POINT | 0 | Cilium treats 0 as TO_LXC, but its&#39;s something we should work to remove. This is intentionally set as unknown, so proto API can guarantee the observation point is always going to be present on trace events. |
-| TO_PROXY | 1 |  |
-| TO_HOST | 2 |  |
-| TO_STACK | 3 |  |
-| TO_OVERLAY | 4 |  |
-| TO_ENDPOINT | 101 | same as TO_LXC, which had a 0 value. This index is intentionally very high so when new segments are added in bpf, there are no collisions |
-| FROM_ENDPOINT | 5 |  |
-| FROM_PROXY | 6 |  |
-| FROM_HOST | 7 |  |
-| FROM_STACK | 8 |  |
-| FROM_OVERLAY | 9 |  |
-| FROM_NETWORK | 10 |  |
-| TO_NETWORK | 11 |  |
+| TO_PROXY | 1 | TO_PROXY indicates network packets are received by l7 proxy. |
```

Comment:
I think this is around the wrong way. Cilium's BPF-based observation points for the Cilium datapath cannot assert that the l7 proxy received the packet, that is up to the l7 proxy. Cilium's BPF observation points can only assert that it _transmitted_ the packets towards the l7 proxy.

```suggestion
| TO_PROXY | 1 | TO_PROXY indicates network packets are transmitted towards the l7 proxy. |
```

If the user configures something that causes drops in between, then Cilium cannot be responsible for this. Otherwise it might look like a bug where Cilium didn't successfully cause the l7 proxy to receive the packet, but in reality Cilium's datapath did its job and the user's external configuration introduces the drop.

Reply:
Likewise, all of the `TO_*` targets are from the perspective of Cilium's datapath. Just before Cilium executes the code to deliver a packet towards a particular destination, it will emit one of the `TO_*` events.